plugins/performance-lab/includes/admin/load.php

```diff
@@ -230,6 +227,24 @@ function perflab_enqueue_features_page_scripts(): void {
 
 	// Enqueue the a11y script.
 	wp_enqueue_script( 'wp-a11y' );
+
+	// Enqueue plugin activate AJAX script and localize script data.
+	wp_enqueue_script(
+		'perflab-plugin-activate-ajax',
+		PERFLAB_PLUGIN_DIR_URL . 'includes/admin/plugin-activate-ajax.js',
```

Comment:
This `PERFLAB_PLUGIN_DIR_URL` is `plugin_dir_url( PERFLAB_MAIN_FILE )`, but I believe here it would be better to use `plugins_url()`:

```suggestion
		plugins_url( 'includes/admin/plugin-activate-ajax.js', PERFLAB_MAIN_FILE ),
```

The reason is that this function allows for filtering as needed.

Reply:
@westonruter I don't think we should randomly change it here though. If not using `plugins_url()` leads to a problem, let's discuss that in a separate issue and implement properly if needed.

Reply:
I've filed https://github.com/WordPress/performance/issues/1647 for this.